src/components/modal/tests/ModalBackdropConnected.spec.tsx

```diff
@@ -109,5 +109,14 @@ describe('ModalBackdrop', () => {
 
       expect(displayProp).toBe(false);
     });
+
+    it('should call onClick when we click on it', () => {
```

Comment:
Tu vas me trouver téteux (:sooxfordcomma:), mais garde un discours neutre / sans première personne.
when we click on it => when clicked